.github/workflows/docker-build-server-harbor.yml

```diff
@@ -0,0 +1,22 @@
+name: Build CAP server image
+
+on:
+  push:
+    branches: next
+    paths-ignore:
+      - "ui/**"
+
+jobs:
+  build_dev:
+    name: Build server image
+    runs-on: ubuntu-latest
+    env:
+      DEBUG: True
+    steps:
+      - name: Build and Export
+        uses: cern-sis/gh-workflows/.github/actions/docker-build@v4.0.0
+        with:
+          image: cern-sis/cap-server-dev
```

Comment:
Why not just `cern-sis/cap-server` Our goal is to use this image in the whole pipeline from dev, qa and prod.